recipes/cloudflared/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "cloudflared" %}
+{% set version = "2022.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cloudflare/cloudflared/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e49db875da7513d8ae950b20c636225016022866850ff3df2484c0f56cdc4bc4
+
+build:
+  script:
+    - make PREFIX="$PREFIX" VERSION={{ version }} MSI_VERSION= install  # [not win]
+    - make PREFIX=%LIBRARY_PREFIX% VERSION={{ version }} MSI_VERSION= install  # [win]
+    - go-licenses save . --save_path=license-files
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go-nocgo') }}
```

You might need to play around with the Go version here.
```suggestion
    - {{ compiler('go-nocgo') }} 1.19
```

---

Thank you!